doc/host-applications.rst

```diff
@@ -197,24 +192,12 @@ Spicy arguments translate into C++ arguments, look at the C++
 prototype that's included for the callback function in the output of
 ``-P``.
 
-A couple more notes on the compilation process for integrating
-Spicy-generated code into custom host applications:
-
-    - Above we used ``spicyc -l`` to link our Spicy code from just a
-      single Spicy source file. If you have more than one source file,
-      you need to link them altogether in a single step. For example,
-      if we had ``A.spicy``, ``B.spicy`` and ``C.spicy``, we'd do::
-
-        # spicyc -c -g A.spicy -o A.cc
-        # spicyc -c -g B.spicy -o B.cc
-        # spicyc -c -g C.spicy -o C.cc
-        # spicyc -l -g A.cc B.cc C.cc -o linker.cc
-        # clang++ A.cc B.cc C.cc linker.cc -o a.out ...
-
-    - If your Spicy code is importing any library modules (e.g., the
-      standard ``filter`` module), you'll need to compile those as
-      well in the same fashion.
+.. note::
 
+    When using ``spicyc -x`` you can specify any Spicy modules you
+    need/want. If any of your Spicy code is importing any library
+    modules (e.g., the standard ``filter`` module), you'll need to add
+    those to the command line as well.
```

Comment:
Is that true? I took the example from https://docs.zeek.org/projects/spicy/en/latest/programming/parsing.html#filters and the following succeeded, even though `foo.spicy` has `import filter`:

```console
spicyc -x foo foo.spicy
$(spicy-config --cxx --cxxflags --ldflags) foo_*cc -o foo
```

Reply:
Good catch. I adapted that from the old text without trying, but we indeed don't need that when building everything at once.